Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -379,6 +379,7 @@ struct PerWebProcessState {
 
 - (void)_textReplacementSession:(NSUUID *)sessionUUID updateState:(WebKit::WebTextReplacementDataState)state forReplacementWithUUID:(NSUUID *)replacementUUID;
 - (void)_removeTextIndicatorStyleForID:(NSUUID *)uuid;
+
```

Comment:
Nit - stray newline (either remove this, or add an additional newline right below the `ENABLE(UNIFIED_TEXT_REPLACEMENT)`)